scripts/Editor/DllManipulatorEditor.cs

```diff
@@ -8,6 +8,8 @@
 #endif
 using System.IO;
 using System;
+using UnityEditor.SceneManagement;
+using UnityEngine.SceneManagement;
```

Comment:
Duplicated?

Reply:
No we need it both from the Editor and Engine. They are not the same.

Reply:
I realized we dont actually need to set the scene as dirty when we set the gui target as dirty. So its been removed. 87dcb2ab6fde2a1a1712ec37ea1277a1b2df138f